src/docfx/build/toc/TableOfContentsLoader.cs

```diff
@@ -289,12 +232,12 @@ private IReadOnlyList<string> GetMonikers(TableOfContentsNode currentItem, List<
 
         private (SourceInfo<string?> resolvedTocHref, TableOfContentsNode? subChildren, TableOfContentsNode? subChildrenFirstItem)
             ProcessTocHref(
-            Document filePath,
-            Document rootPath,
-            List<Document> referencedFiles,
-            List<Document> referencedTocs,
-            SourceInfo<string?> tocHref,
-            List<Error> errors)
+                Document filePath,
+                Document rootPath,
+                List<Document> referencedFiles,
+                List<Document> referencedTocs,
+                SourceInfo<string?> tocHref,
+                List<Error> errors)
```

Comment:
do we want to return `List<Error>` instead of passing it along?

Reply:
return value put a huge burden on the caller, so I tend to prefer putting it as and argument. If we have a stack only `ErrorBuilder`, it will also be an argument.